scripts/sync-pkgs-wireit.js

```diff
@@ -115,6 +126,10 @@ const updateLibPkgs = async ({ libPkgs }) => {
       // Webpack depends on ESM output from other packages.
       addDeps("build:dist:dev", dep, `../${dep}:build:lib:esm`);
       addDeps("build:dist:min", dep, `../${dep}:build:lib:esm`);
+
+      // TypeScript checking depends on types output from other packages.
+      addDeps("types:check", dep, `../${dep}:types:create`);
```

Comment:
Need to split CJS, ESM

Reply:
So here's what I decide to do...
The esm and cjs builds are identical, so I consolidated to a single `types:create` script which will compile to the `es` folder, then copy to the `lib` folder. It's faster and way simpler.

Reply:
I think that sounds fine. All the `build:lib:esm` and `build:lib:cjs` and `build:dist:*` tasks can just depend on `types:create`.

When you've pushed the change, we should both review the full `victory-core` task for that one `outputs`-wise, etc. to make sure that we don't clobber into the `:cjs` scripts...